cranelift/jit/Cargo.toml

```diff
@@ -19,6 +19,7 @@ cranelift-native = { workspace = true }
 cranelift-codegen = { workspace = true, features = ["std"] }
 cranelift-entity = { workspace = true }
 cranelift-control = { workspace = true }
+wasmtime-unwinder = { workspace = true, features = ["cranelift"] }
```

Comment:
Please make this an optional dependency. I don't want to pull this in for cg_clif.